neurom/io/datawrapper.py

```diff
@@ -271,3 +271,10 @@ def get_datawrapper(self, file_format='BlockNeuronBuilder', data_wrapper=DataWra
         self._check_consistency()
         datablock, sections = self._make_datablock()
         return data_wrapper(datablock, file_format, sections)
+
+    def __str__(self):
+        '''pretty print neuron'''
+        ret = []
+        for section_id, section in self.sections.items():
+            ret.append(str(section_id) + ' ' + str(section))
```

Comment:
No love for list comprehensions? Weird :)
["{0} {1}".format(kv) for kv in dict.items()]